### ISSUE TITLE
make gettext: 'title' object has no attribute 'uid'

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#566]
I'm using Sphinx tip, Python 2.7.1, Docutils 0.7, Jinja 2.5.5

```
#!pytb

Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/sphinx/cmdline.py", line 173, in main
    app.build(force_all, filenames)
  File "/usr/lib/python2.7/site-packages/sphinx/application.py", line 203, in build
    self.builder.build_update()
  File "/usr/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 193, in build_update
    'out of date' &#37; len(to_build))
  File "/usr/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 249, in build
    self.write(docnames, list(updated_docnames), method)
  File "/usr/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 288, in write
    self.write_doc(docname, doctree)
  File "/usr/lib/python2.7/site-packages/sphinx/builders/intl.py", line 70, in write_doc
    self.handle_versioning(docname, doctree, nodes.TextElement)
  File "/usr/lib/python2.7/site-packages/sphinx/builders/versioning.py", line 64, in handle_versioning
    list(merge_doctrees(old_doctree, doctree, condition))
  File "/usr/lib/python2.7/site-packages/sphinx/versioning.py", line 64, in merge_doctrees
    new_node.uid = old_node.uid
AttributeError: 'title' object has no attribute 'uid'
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/566
- Originally reported by: Anonymous
- Originally created at: 2010-12-07T17:51:57.085

**Comment by shimizukawa** [comment]
_From Robert Lehmann on 2010-12-10 07:03:43+00:00_

What documents are you trying to build (Sphinx docs or your own project)?  Did you try cleaning all caches (_build/doctrees comes to mind)?

**Comment by shimizukawa** [comment]
_From Anonymous on 2010-12-10 14:26:32+00:00_

Removing _build/doctrees fixed it.

However if I use `make html` first and then `make gettext` the error appears again. The other way round works. The doctrees are different in either case (dunno whether that's intentional though).

**Comment by shimizukawa** [comment]
_From Anonymous on 2010-12-10 14:28:11+00:00_

forgot to mention that this are my own docs I unfortunately can't publish.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-01-08 16:33:00+00:00_

Fixed in <<cset c266128c6d0f06f4f11c34ddc6e533fd2c425ccc>>; gettext builder now makes sure that it gets its own environment.
